ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -389,6 +389,19 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-sno
+- artifact_dir: /tmp/artifacts
+  as: e2e-aws-fips
+  steps:
+    cluster_profile: aws
+    env:
+      FIPS_ENABLED: "true"
+      TEST_SUITE: openshift/conformance/parallel
+    pre:
+    - chain: ipi-aws-pre
+    - ref: fips-check
+    test:
+    - ref: openshift-e2e-test
+    workflow: ipi-aws
```

Comment:
Technically this is even the `openshift-e2e-test` wortkflow and you do not need to override its `test`, just `pre`.

/hold